tests/unit/test_extensions.py

```diff
@@ -40,3 +42,21 @@ def test_error_handler(crawler):
     assert 'errors' in crawler.spider.state
     assert crawler.spider.state['errors'][0]["exception"] == "Some failure"
     assert crawler.spider.state['errors'][0]["sender"] == response
+
+
+@mock.patch("hepcrawl.extensions.sentry_sdk.init")
+def test_sentry_logging_init(mock_sentry_sdk, crawler):
+    """Test SentryLogging extension."""
+    log_settings = {
+        "SENTRY_DSN" : "TEST_SENTRY_DSN",
+        "EXTENSIONS" : {
+            'hepcrawl.extensions.SentryLogging': 100,
+            'scrapy_sentry.extensions.Errors': 200,
+            'hepcrawl.extensions.ErrorHandler': 300,
```

Comment:
It's not already in project settings? Or are settings for tests different?

Reply:
In `settings.py`, it is set only if environment variable `APP_SENTRY_DSN` exists which defined in kubernetes. So I had to define them seperately in tests.